PSReadLine/KeyBindings.cs

```diff
@@ -251,6 +251,9 @@ void SetDefaultWindowsBindings()
                 _dispatchTable.Add(Keys.PageDown,     MakeKeyHandler(ScrollDisplayDown,     "ScrollDisplayDown"));
                 _dispatchTable.Add(Keys.CtrlPageUp,   MakeKeyHandler(ScrollDisplayUpLine,   "ScrollDisplayUpLine"));
                 _dispatchTable.Add(Keys.CtrlPageDown, MakeKeyHandler(ScrollDisplayDownLine, "ScrollDisplayDownLine"));
+
+                _dispatchTable.Add(Keys.AltH, MakeKeyHandler(DynamicHelpParameter, "DynamicHelpParameter"));
+                _dispatchTable.Add(Keys.F1,   MakeKeyHandler(DynamicHelpFullHelp,  "DynamicHelpFullHelp"));
```

Comment:
Dynamic help should have default bindings in Emacs and VI edit modes too.

Reply:
fixed